static/app/components/group/externalIssueForm.tsx

```diff
@@ -69,6 +72,14 @@ export default class ExternalIssueForm extends AbstractExternalIssueForm<Props,
   onSubmitSuccess = (_data: IntegrationExternalIssue): void => {
     const {onChange, closeModal} = this.props;
     const {action} = this.state;
+
+    trackAdvancedAnalyticsEvent('issue_details.external_issue_created', {
+      organization: this.props.organization,
+      ...getAnalyticsDataForGroup(this.props.group),
+      external_issue_provider: this.props.integration.provider.key,
+      external_issue_type: 'integration',
```

Comment:
@malwilley can we call it `first_party`? That's the standard we follow in other places. See `IntegrationType`

Reply:
Thanks for pointing this out - didn't know about that! Changed to `IntegrationType`

Out of curiosity what's the difference between `first_party` and `sentry_app`?

Reply:
@malwilley first_party means we built it, sentry_app means it's on the integration platform and a 3rd party built it